README.md

```diff
@@ -138,6 +138,9 @@ Thanks for the [ansi\_term crate](https://github.com/ogham/rust-ansi-term) for
 providing a reference implementation, which greatly helped making this crate
 output correct strings.
 
+## Minimum Supported Rust Version (MSRV)
+The current MSRV is `1.63`, which is checked and enforced automatically via CI. This version may change in the future in minor version bumps, so if you require a specific Rust version you should pin the crate to an exact version.
```

Comment:
```suggestion
The current MSRV is `1.63`, which is checked and enforced automatically via CI. This version may change in the future in minor version bumps, so if you require a specific Rust version you should use a restricted version requirement such as `~X.Y`.
```
Feel free to reword, but `exact` sound like you'd need to pin to a specific patch version.

Reply:
I was hinting at an exact version, but having patch versions allowed is definitely preferred.